prody/proteins/interactions.py

```diff
@@ -0,0 +1,1109 @@
+    Predictions for proteins only. 
+    To compute protein-ligand interactions use calcLigandInteractions() or define **kwargs"""
+
+    try:
+        coords = (atoms._getCoords() if hasattr(atoms, '_getCoords') else
+                    atoms.getCoords())
+    except AttributeError:
+        try:
+            checkCoords(coords)
+        except TypeError:
+            raise TypeError('coords must be an object '
+                            'with `getCoords` method')
+
+    aromatic_dic = {'TRP':'noh and not backbone and not name CB NE1 CD1 CG',
+                'PHE':'noh and not backbone and not name CB',
+                'TYR':'noh and not backbone and not name CB and not name OH',
+                'HIS':'noh and not backbone and not name CB'}
+    
+    for key, value in kwargs.items():
+        aromatic_dic[key] = value
+    
+    atoms_cylic = atoms.select('resname TRP PHE TYR HIS')
+    aromatic_resids = list(set(zip(atoms_cylic.getResnums(), atoms_cylic.getChids())))
+
+    LOGGER.info('Calculating Pi stacking interactions.')
+    PiStack_calculations = []
+    for i in aromatic_resids:
+        for j in aromatic_resids:
+            if i != j: 
+                sele1_name = atoms.select('resid '+str(i[0])+' and chain '+i[1]+' and name CA').getResnames()
+                sele1 = atoms.select('resid '+str(i[0])+' and chain '+i[1]+' and '+aromatic_dic[sele1_name[0]])
+                
+                sele2_name = atoms.select('resid '+str(j[0])+' and chain '+j[1]+' and name CA').getResnames()
+                sele2 = atoms.select('resid '+str(j[0])+' and chain '+j[1]+' and '+aromatic_dic[sele2_name[0]])
+                
+                if sele1 != None and sele2 != None:
+                    a1, b1, c1, a2, b2, c2 = calcPlane(sele1)[:3]+calcPlane(sele2)[:3]
+                    RingRing_angle = calcAngleBetweenPlanes(a1, b1, c1, a2, b2, c2) # plane is computed based on 3 points of rings           
+                    RingRing_distance = calcDistance(calcCenter(sele1.getCoords()),calcCenter(sele2.getCoords()))
+                    if RingRing_distance < distA and angle_min < RingRing_angle < angle_max:
+                        PiStack_calculations.append([sele1_name[0]+str(sele1.getResnums()[0]), '_'.join(map(str,sele1.getIndices())), sele1.getChids()[0],
+                                                     sele2_name[0]+str(sele2.getResnums()[0]), '_'.join(map(str,sele2.getIndices())), sele2.getChids()[0],
+                                                     round(RingRing_distance,3), round(RingRing_angle,3)])
+    
+    PiStack_calculations = sorted(PiStack_calculations, key=lambda x : x[-2])   
+    LOGGER.info("Number of detected Pi stacking interactions: {0}.".format(len(PiStack_calculations)))
+    
+    return PiStack_calculations
+
+
+def calcPiCation(atoms, distA=5.0, extraSele=None, **kwargs):
+    """Searching for cation-Pi interaction i.e. between aromatic ring and positively charged residue (ARG and LYS)
```

Comment:
I don't think searching is the right form of the verb. You could put search as a command or searches as a description.